runner/Makefile

```diff
@@ -1,9 +1,10 @@
-NAME ?= summerwind/actions-runner
-DIND_RUNNER_NAME ?= ${NAME}-dind
+DOCKER_USER ?= summerwind
+RUNNER_NAME ?= ${DOCKER_USER}/actions-runner
```

Comment:
We'd need to update this part of your contribution guide as well

<img width="1092" alt="CleanShot 2021-12-12 at 15 22 37@2x" src="https://user-images.githubusercontent.com/22009/145702678-dbb09a78-9d22-4c08-96b6-7da6ef7e49a2.png">

Reply:
Addressed in aab3fae